CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## 1.0.5
+
+### Features
```

Comment:
Can we add this, as it sneaked in with https://github.com/nextcloud/whiteboard/commit/b034156e6acb28218bee48ca4bb332ae9685c185 ;)

> - Compatibility with Nextcloud 31